### ISSUE TITLE
USER STORY: Model optimization V7

### ISSUE THREAD

**Issue by Huddy2022** [Huddy2022/milestone-project-mildew-detection-in-cherry-leaves#26]
As a **data scientist** I want to **train the CNN model using the pre-processed dataset with re-sizes images 100 x 100 and optimize the hyperparameters resorting back to V2 model but tweaking filters 32, 64 & 128, dropout 0.4, remove regularizers L1, L2 & none but keep batchnormalization to achieve the best performance,** so that **the model can accurately classify cherry leaves as healthy or containing powdery mildew.**
